pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -217,3 +222,61 @@ The Tracking Plan will be delivered with a `POST` request containing the Trackin
+
+#### Nested Event Model
+
+| Attribute | Type       | Description |
+| :-------- | :--------- | :---------- |
+| eventId   | string     | Event id    |
+| where     | array(???) | ???         |
+| groupBy   | array(???) | ???         |
+
+#### Source Model
+
+| Attribute           | Type          | Description                                                   |
+| :------------------ | :------------ | :------------------------------------------------------------ |
+| id                  | string        | Source id                                                     |
+| name                | string        | Source name                                                   |
+| programmingLanguage | string        | Name of programming language (e.g. "Javascript", "Python")    |
+| developmentPlatform | string        | Name of development platroft platform (e.g. "Web", "Android") |
+| destinations        | array(string) | Destination ids where this source sends data to               |
+
+#### Destination Model
+
+| Attribute | Type   | Description                                                                                                        |
+| :-------- | :----- | :----------------------------------------------------------------------------------------------------------------- |
+| id        | string | Destination id                                                                                                     |
+| name      | string | Destination name                                                                                                   |
+| type      | string | Name of the provider (e.g. "Segment", "Mispanel") where the data is sent to if managed by Avo, "Custom" otherwise. |
+
+#### Category Model
+
+| Attribute   | Type          | Description                        |
+| :---------- | :------------ | :--------------------------------- |
+| id          | string        | Category id                        |
+| name        | string        | Category name                      |
+| description | string        | Category description               |
+| metrics     | array(string) | Metric ids belonging this category |
+| events      | array(string) | Event ids belonging this category  |
+
+#### PublishInfo Model
+
+| Attribute       | Type             | Description                                                               |
+| :-------------- | :--------------- | :------------------------------------------------------------------------ |
+| branchId        | id               | Branch id                                                                 |
+| branchName      | string           | Branch name                                                               |
+| integrationId   | string           | ???                                                                       |
+| integrationName | string           | ???                                                                       |
```

Comment:
@bjornj12 could you help with fields marked with `???`

Reply:
```suggestion
| integrationName | string           | Name of the publish integration (seen in Tracking Plan -> publishing -> <IntegrationName>                                                                      |
```